.circleci/config.yml

```diff
@@ -1,32 +1,64 @@
 version: 2.1
 jobs:
   build:
+    parameters:
+      build-type:
+        type: string
+    docker:
+      - image: rockylinux:8
+    steps:
+      - checkout
+      - run:
+          name: Install System Dependencies
+          command: ./tests/ci/setup.sh build
+      - run:
+          name: Build Software Package
+          command: ./tests/ci/build.sh << parameters.build-type >>
+      - persist_to_workspace:
+          root: .
+          paths:
+            - dist/*
+
+  test:
     parameters:
       install-type:
         type: string
     docker:
-      # The first image entry here will be used as the image for the parent container.
       - image: tools-ext-01.ccr.xdmod.org/xdmod-job_performance-10.0.0:rockylinux8-0.1
     environment:
       TERM: xterm
+      TERMINFO: /bin/bash
       COMPOSER_ALLOW_SUPERUSER: 1
       XDMOD_REALMS: 'jobs,storage,cloud'
       XDMOD_IS_CORE: yes
       XDMOD_INSTALL_DIR: /xdmod
       XDMOD_TEST_MODE: << parameters.install-type >>
     steps:
       - checkout
-      - run:
-          name: Install System Dependencies
-          command: ./tests/ci/setup.sh
       - run:
           name: Create Test Result Directories
           command: |
             mkdir -p shippable/testresults
             mkdir -p shippable/codecoverage
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install Docker Compose
+          command: |
+            dnf install -y dnf-utils
+            dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+            dnf install -y docker-ce docker-ce-cli docker-compose-plugin
+
+      - setup_remote_docker
+      - run:
+          name: Build services
+          command: docker compose -f ./tests/ci/srv/services.yml build
+      - run:
+          name: Start services
+          command: docker compose -f ./tests/ci/srv/services.yml up -d
       - run:
           name: Run Bootstrap
-          command: ./tests/integration_tests/bootstrap.sh
+          command: ./tests/ci/test/bootstrap.sh $XDMOD_TEST_MODE
```

Comment:
The bootstrap.sh script does not appear to check its commandline arguments. Please double check this change